### DISCUSSION TITLE
第二弹 - Data Source | Databend 内幕大揭秘

### DISCUSSION THREAD

**Discussion by giscus** [PsiACE/databend-internals#43] [Announcements]
# 第二弹 - Data Source | Databend 内幕大揭秘

minibend 系列的第二篇，主要讲 Data Source 相关的一些内容。minibend：从零开始、使用 Rust 构建的查询引擎。

https://psiace.github.io/databend-internals/docs/minibend/datasource/

**Comment by xiangel** [comment]
请问这个系列还更新么？
